ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java

```diff
@@ -11,7 +11,7 @@
  * distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
This class is package-private but has a main method. I guess this is valid, and the main method is accessible, but it seems a bit odd.